frontend/src/metabase/containers/DataPicker/tests/DataPicker-RawData.unit.spec.ts

```diff
@@ -163,21 +163,22 @@ describe("DataPicker — picking raw data", () => {
     });
 
     it("resets selected tables on schema change", async () => {
-      const [schema1, schema2] = MULTI_SCHEMA_DATABASE.schemas;
-      const [schema1Table] = schema1.tables;
+      const schema1Table = SAMPLE_TABLE_3;
+      const schema1 = SAMPLE_TABLE_3.schema;
+      const schema2 = SAMPLE_TABLE_4.schema;
```

Comment:
nit: I actually like how we used to destructure tables from a database before. That tells that those tables belong to the same DB

comparing this:
```ts
const [table1, table2] = MULTI_SCHEMA_DATABASE.tables // deal with nullable `tables` somehow
```
vs this change. It isn't quite apparent that `SAMPLE_TABLE_3` and `SAMPLE_TABLE_4` belongs to the same database.